CHANGELOG.md

```diff
@@ -15,263 +15,263 @@ all APIs might be changed.
 
 ### Bug Fixes
 
-- Fixed some compile errors when `ws_stream_wasm` was enabled and `tungstenite`
-  was not ([#111](https://github.com/obmarg/graphql-ws-client/pull/111))
+-   Fixed some compile errors when `ws_stream_wasm` was enabled and `tungstenite`
```

Comment:
Can you revert the formatting changes in this and all the other markdown files please.